src/auto-start.ts

```diff
@@ -39,6 +39,11 @@ if (typeof document !== 'undefined' && typeof window !== 'undefined') {
 			Prism.highlightAll();
 		}
 	});
+
+	// Make Prism available globally
+	if (typeof globalThis !== 'undefined') {
+		(globalThis as any).Prism = Prism;
+	}
```

Comment:
Even though I included this chunk in the PR, I wonder if we want to expose the Prism instance globally. 🤔

Reply:
We don’t, but the iife version should!

Reply:
> We don’t, but the iife version should!

This is the issue with Rollup I still can't solve. It doesn't work as expected. I might miss something. I'll give it another look.

> You can check if `document.currentScript` is defined and expose it as a global if so. This means that the script is included via a non-module script and that it's in a browser, not Node.

Oh, will try